src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public enum Colors {
```

Comment:
enum class names should be in the singular form as per the style guide
```suggestion
public enum Color {
```